2023-03-30 10:30:16,709 ERROR [mpl] [apps.py:14] - test
2023-03-30 10:30:41,265 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 10:30:41,266 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 10:30:41,266 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 10:30:52,819 ERROR [mpl] [<input>:1] - 123
2023-03-30 10:32:10,023 ERROR [mpl] [<input>:1] - 123
2023-03-30 10:32:34,760 ERROR [mpl] [apps.py:14] - test
2023-03-30 10:32:35,291 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 10:32:35,291 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 10:32:35,292 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 10:33:15,351 ERROR [mpl] [apps.py:14] - test
2023-03-30 10:33:15,522 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 10:33:15,522 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 10:33:15,523 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 10:33:33,935 ERROR [mpl] [<input>:1] - 123
2023-03-30 10:34:15,820 ERROR [mpl] [apps.py:14] - test
2023-03-30 10:34:15,981 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 10:34:15,982 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 10:34:15,983 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 10:34:36,296 ERROR [mpl] [<input>:1] - 123
2023-03-30 10:36:59,095 ERROR [mpl] [apps.py:14] - test
2023-03-30 10:36:59,205 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 10:36:59,205 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 10:36:59,206 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 10:37:18,958 ERROR [mpl] [apps.py:14] - test
2023-03-30 10:37:19,046 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 10:37:19,047 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 10:37:19,047 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 10:38:11,148 ERROR [mpl] [apps.py:14] - test
2023-03-30 10:38:11,239 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 10:38:11,240 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 10:38:11,240 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 10:38:25,165 ERROR [mpl] [MCPWorker.py:56] - FdcDataGroup matching query does not exist.
2023-03-30 10:38:25,167 ERROR [mpl] [MCPWorker.py:57] - None
2023-03-30 10:39:30,401 ERROR [mpl] [apps.py:14] - test
2023-03-30 10:39:30,497 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 10:39:30,498 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 10:39:30,498 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 10:39:39,017 ERROR [mpl] [MCPWorker.py:56] - FdcDataGroup matching query does not exist.
2023-03-30 10:39:39,019 ERROR [mpl] [MCPWorker.py:57] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 52, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 32, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 57, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 10:46:15,692 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 10:46:15,693 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 10:46:15,693 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 10:47:10,391 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 10:47:10,391 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 10:47:10,392 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 10:51:28,527 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 10:51:28,528 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 10:51:28,528 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 10:59:20,064 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 10:59:20,065 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 10:59:20,066 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:02:24,408 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 11:02:24,408 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 11:02:24,409 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:03:24,424 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 11:03:24,425 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 11:03:24,426 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:03:58,145 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 11:03:58,146 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 11:03:58,147 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:04:44,139 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 11:04:44,140 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 11:04:44,140 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:05:09,448 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 11:05:09,449 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 11:05:09,450 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:06:02,952 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 11:06:02,952 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 11:06:02,953 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:07:08,102 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 11:07:08,102 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 11:07:08,103 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:07:25,513 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 11:07:25,514 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 11:07:25,514 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:08:11,300 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 11:08:11,301 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 11:08:11,301 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:08:52,376 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 11:08:52,377 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 11:08:52,378 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:17:01,033 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 11:17:01,034 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 11:17:01,034 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:18:16,893 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 11:18:16,893 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 11:18:16,894 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:20:13,662 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 11:20:13,662 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 11:20:13,663 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:20:37,766 ERROR [mpl] [MCPWorker.py:55] - FdcDataGroup matching query does not exist.
2023-03-30 11:20:39,191 ERROR [mpl] [MCPWorker.py:56] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 52, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 32, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 56, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 11:22:11,307 ERROR [mpl] [MCPWorker.py:55] - FdcDataGroup matching query does not exist.
2023-03-30 11:22:11,317 ERROR [mpl] [MCPWorker.py:56] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 52, in mplPWorker\n    except Exception as e:\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 32, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 56, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 11:23:31,393 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 11:23:31,393 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 11:23:31,394 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:23:47,009 ERROR [mpl] [MCPWorker.py:55] - FdcDataGroup matching query does not exist.
2023-03-30 11:23:47,011 ERROR [mpl] [MCPWorker.py:56] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 51, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 32, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 56, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 11:24:57,969 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 11:24:57,969 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 11:24:57,970 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:27:32,872 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 11:27:32,873 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 11:27:32,873 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:29:24,297 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 11:29:24,298 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 11:29:24,298 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 11:44:33,468 ERROR [mpl] [MPListener.py:75] - on_receiver_loop_completed = None
2023-03-30 12:44:42,544 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 12:44:42,546 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 12:44:42,546 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 13:09:47,105 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 13:09:47,106 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 13:09:47,107 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 13:27:54,681 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 13:27:54,682 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 13:27:54,682 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 13:29:33,260 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 13:29:33,261 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 13:29:33,261 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 13:35:27,473 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 13:35:27,474 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 13:35:27,474 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 13:37:58,651 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 13:37:58,652 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 13:37:58,652 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 13:51:14,151 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 13:51:14,152 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 13:51:14,153 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 14:08:15,734 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 14:08:15,735 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 14:08:15,735 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 14:23:45,752 ERROR [mpl] [MPListener.py:75] - on_receiver_loop_completed = None
2023-03-30 14:32:15,059 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 14:32:15,061 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 14:32:15,062 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 14:33:05,972 ERROR [mpl] [MCPWorker.py:60] - 'TEST'
2023-03-30 14:33:05,976 ERROR [mpl] [MCPWorker.py:61] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 51, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 32, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 61, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 14:33:51,190 ERROR [mpl] [MCPWorker.py:60] - 'TEST'
2023-03-30 14:33:51,197 ERROR [mpl] [MCPWorker.py:61] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 51, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 32, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 61, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 14:34:07,668 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 14:34:07,669 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 14:34:07,669 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 14:34:16,346 ERROR [mpl] [MCPWorker.py:60] - 'TEST'
2023-03-30 14:34:16,355 ERROR [mpl] [MCPWorker.py:61] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 51, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 32, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 61, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 14:34:29,244 ERROR [mpl] [MCPWorker.py:60] - 'TEST'
2023-03-30 14:34:29,246 ERROR [mpl] [MCPWorker.py:61] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 51, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 32, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 61, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 14:35:20,769 ERROR [mpl] [MCPWorker.py:60] - 'TEST'
2023-03-30 14:35:20,775 ERROR [mpl] [MCPWorker.py:61] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 51, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 32, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 61, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 14:39:42,934 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 14:39:42,935 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 14:39:42,935 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 14:41:11,417 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 14:41:11,417 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 14:41:11,418 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 14:57:51,390 ERROR [mpl] [MPListener.py:75] - on_receiver_loop_completed = None
2023-03-30 15:21:31,081 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 15:21:31,081 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 15:21:31,082 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 15:36:31,262 ERROR [mpl] [MPListener.py:75] - on_receiver_loop_completed = None
2023-03-30 15:53:39,178 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
2023-03-30 15:53:39,179 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCMessageCore1', 'id': '1_message', 'ack': 'auto'}],body=}
2023-03-30 15:53:39,180 ERROR [mpl] [MPListener.py:68] - on_send = {cmd=SUBSCRIBE,headers=[{'destination': '/queue/FDCCommandCore1', 'id': '1_command', 'ack': 'auto'}],body=}
2023-03-30 16:04:15,067 ERROR [mpl] [MPListener.py:75] - on_receiver_loop_completed = None
2023-03-30 16:08:38,319 ERROR [mpl] [MPListener.py:64] - 'EqpCode'
2023-03-30 16:08:38,790 ERROR [mpl] [MPListener.py:65] - ['  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydev_bundle\\pydev_monkey.py", line 795, in __call__\n    ret = self.original_func(*self.args, **self.kwargs)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 995, in _bootstrap\n    self._bootstrap_inner()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 1038, in _bootstrap_inner\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 975, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 341, in __receiver_loop\n    self.process_frame(f, frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 190, in process_frame\n    self.notify(frame_type, f)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 243, in notify\n    notify_func(frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Listener\\MPListener.py", line 65, in on_message\n    self.__logger.error(traceback.format_stack())\n']
2023-03-30 16:08:40,926 ERROR [mpl] [MPListener.py:64] - 'EqpCode'
2023-03-30 16:08:41,266 ERROR [mpl] [MPListener.py:65] - ['  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydev_bundle\\pydev_monkey.py", line 795, in __call__\n    ret = self.original_func(*self.args, **self.kwargs)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 995, in _bootstrap\n    self._bootstrap_inner()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 1038, in _bootstrap_inner\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 975, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 341, in __receiver_loop\n    self.process_frame(f, frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 190, in process_frame\n    self.notify(frame_type, f)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 243, in notify\n    notify_func(frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Listener\\MPListener.py", line 65, in on_message\n    self.__logger.error(traceback.format_stack())\n']
2023-03-30 16:09:11,738 ERROR [mpl] [MPListener.py:64] - 'EqpCode'
2023-03-30 16:09:11,761 ERROR [mpl] [MPListener.py:65] - ['  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydev_bundle\\pydev_monkey.py", line 795, in __call__\n    ret = self.original_func(*self.args, **self.kwargs)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 995, in _bootstrap\n    self._bootstrap_inner()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 1038, in _bootstrap_inner\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 975, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 341, in __receiver_loop\n    self.process_frame(f, frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 190, in process_frame\n    self.notify(frame_type, f)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 243, in notify\n    notify_func(frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Listener\\MPListener.py", line 65, in on_message\n    self.__logger.error(traceback.format_stack())\n']
2023-03-30 16:49:36,792 ERROR [mpl] [MCPWorker.py:62] - 'djongo' isn't an available database backend or couldn't be imported. Check the above exception. To use one of the built-in backends, use 'django.db.backends.XXX', where XXX is one of:
    'mysql', 'oracle', 'postgresql', 'sqlite3'
2023-03-30 16:49:36,801 ERROR [mpl] [MCPWorker.py:63] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 52, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 33, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 63, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 16:50:59,568 ERROR [mpl] [MCPWorker.py:62] - 'djongo' isn't an available database backend or couldn't be imported. Check the above exception. To use one of the built-in backends, use 'django.db.backends.XXX', where XXX is one of:
    'mysql', 'oracle', 'postgresql', 'sqlite3'
2023-03-30 16:50:59,573 ERROR [mpl] [MCPWorker.py:63] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 52, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 33, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 63, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 16:51:52,037 ERROR [mpl] [MCPWorker.py:62] - 'djongo' isn't an available database backend or couldn't be imported. Check the above exception. To use one of the built-in backends, use 'django.db.backends.XXX', where XXX is one of:
    'mysql', 'oracle', 'postgresql', 'sqlite3'
2023-03-30 16:51:52,038 ERROR [mpl] [MCPWorker.py:63] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 52, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 33, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 63, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 16:52:21,086 ERROR [mpl] [MCPWorker.py:62] - 'djongo' isn't an available database backend or couldn't be imported. Check the above exception. To use one of the built-in backends, use 'django.db.backends.XXX', where XXX is one of:
    'mysql', 'oracle', 'postgresql', 'sqlite3'
2023-03-30 16:52:21,090 ERROR [mpl] [MCPWorker.py:63] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 52, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 33, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 63, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 16:53:06,528 ERROR [mpl] [MCPWorker.py:62] - 'djongo' isn't an available database backend or couldn't be imported. Check the above exception. To use one of the built-in backends, use 'django.db.backends.XXX', where XXX is one of:
    'mysql', 'oracle', 'postgresql', 'sqlite3'
2023-03-30 16:53:06,536 ERROR [mpl] [MCPWorker.py:63] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 52, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 33, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 63, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 16:55:53,316 ERROR [mpl] [MCPWorker.py:62] - 'djongo' isn't an available database backend or couldn't be imported. Check the above exception. To use one of the built-in backends, use 'django.db.backends.XXX', where XXX is one of:
    'mysql', 'oracle', 'postgresql', 'sqlite3'
2023-03-30 16:55:53,322 ERROR [mpl] [MCPWorker.py:63] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 52, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 33, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 63, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 16:58:26,278 ERROR [mpl] [MCPWorker.py:62] - 'djongo' isn't an available database backend or couldn't be imported. Check the above exception. To use one of the built-in backends, use 'django.db.backends.XXX', where XXX is one of:
    'mysql', 'oracle', 'postgresql', 'sqlite3'
2023-03-30 16:58:26,283 ERROR [mpl] [MCPWorker.py:63] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 52, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 33, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 63, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 16:58:51,618 ERROR [mpl] [MCPWorker.py:62] - 'djongo' isn't an available database backend or couldn't be imported. Check the above exception. To use one of the built-in backends, use 'django.db.backends.XXX', where XXX is one of:
    'mysql', 'oracle', 'postgresql', 'sqlite3'
2023-03-30 16:58:51,620 ERROR [mpl] [MCPWorker.py:63] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 52, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 33, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 63, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 17:00:03,867 ERROR [mpl] [MCPWorker.py:62] - 'djongo' isn't an available database backend or couldn't be imported. Check the above exception. To use one of the built-in backends, use 'django.db.backends.XXX', where XXX is one of:
    'mysql', 'oracle', 'postgresql', 'sqlite3'
2023-03-30 17:00:03,872 ERROR [mpl] [MCPWorker.py:63] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 52, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 33, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 63, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 17:14:26,089 ERROR [mpl] [MCPWorker.py:62] - 'djongo' isn't an available database backend or couldn't be imported. Check the above exception. To use one of the built-in backends, use 'django.db.backends.XXX', where XXX is one of:
    'mysql', 'oracle', 'postgresql', 'sqlite3'
2023-03-30 17:16:44,955 ERROR [mpl] [MCPWorker.py:62] - 'djongo' isn't an available database backend or couldn't be imported. Check the above exception. To use one of the built-in backends, use 'django.db.backends.XXX', where XXX is one of:
    'mysql', 'oracle', 'postgresql', 'sqlite3'
2023-03-30 17:16:44,959 ERROR [mpl] [MCPWorker.py:63] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 52, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 33, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 63, in run\n    logger.error(traceback.format_stack())\n']
2023-03-30 17:17:07,159 ERROR [mpl] [MCPWorker.py:62] - 'djongo' isn't an available database backend or couldn't be imported. Check the above exception. To use one of the built-in backends, use 'django.db.backends.XXX', where XXX is one of:
    'mysql', 'oracle', 'postgresql', 'sqlite3'
2023-03-30 17:17:07,165 ERROR [mpl] [MCPWorker.py:63] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 52, in mplPWorker\n    mplWorker.messageParser(message)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLWorker.py", line 33, in messageParser\n    self.__mcpWorker.run(self.__module, self.__context)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mcp\\Process\\MCPWorker.py", line 63, in run\n    logger.error(traceback.format_stack())\n']
2023-03-31 13:15:19,775 ERROR [mpl] [MPListener.py:66] - 'EqpCode'
2023-03-31 13:15:19,792 ERROR [mpl] [MPListener.py:67] - ['  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydev_bundle\\pydev_monkey.py", line 795, in __call__\n    ret = self.original_func(*self.args, **self.kwargs)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 995, in _bootstrap\n    self._bootstrap_inner()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 1038, in _bootstrap_inner\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 975, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 341, in __receiver_loop\n    self.process_frame(f, frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 190, in process_frame\n    self.notify(frame_type, f)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 243, in notify\n    notify_func(frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Listener\\MPListener.py", line 67, in on_message\n    self.__logger.error(traceback.format_stack())\n']
2023-03-31 15:19:02,343 ERROR [mpl] [MPListener.py:55] - FdcEqpModuleResDto.__init__() got an unexpected keyword argument 'detail'
2023-03-31 15:19:02,374 ERROR [mpl] [MPListener.py:56] - ['  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydev_bundle\\pydev_monkey.py", line 795, in __call__\n    ret = self.original_func(*self.args, **self.kwargs)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 995, in _bootstrap\n    self._bootstrap_inner()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 1038, in _bootstrap_inner\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 975, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 341, in __receiver_loop\n    self.process_frame(f, frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 190, in process_frame\n    self.notify(frame_type, f)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 243, in notify\n    notify_func(frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Listener\\MPListener.py", line 56, in on_message\n    self.__logger.error(traceback.format_stack())\n']
2023-03-31 15:19:03,714 ERROR [mpl] [MPListener.py:55] - FdcEqpModuleResDto.__init__() got an unexpected keyword argument 'detail'
2023-03-31 15:19:03,716 ERROR [mpl] [MPListener.py:56] - ['  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydev_bundle\\pydev_monkey.py", line 795, in __call__\n    ret = self.original_func(*self.args, **self.kwargs)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 995, in _bootstrap\n    self._bootstrap_inner()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 1038, in _bootstrap_inner\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 975, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 341, in __receiver_loop\n    self.process_frame(f, frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 190, in process_frame\n    self.notify(frame_type, f)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 243, in notify\n    notify_func(frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Listener\\MPListener.py", line 56, in on_message\n    self.__logger.error(traceback.format_stack())\n']
2023-03-31 15:19:05,106 ERROR [mpl] [MPListener.py:55] - FdcEqpModuleResDto.__init__() got an unexpected keyword argument 'detail'
2023-03-31 15:19:05,106 ERROR [mpl] [MPListener.py:56] - ['  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydev_bundle\\pydev_monkey.py", line 795, in __call__\n    ret = self.original_func(*self.args, **self.kwargs)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 995, in _bootstrap\n    self._bootstrap_inner()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 1038, in _bootstrap_inner\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 975, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 341, in __receiver_loop\n    self.process_frame(f, frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 190, in process_frame\n    self.notify(frame_type, f)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 243, in notify\n    notify_func(frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Listener\\MPListener.py", line 56, in on_message\n    self.__logger.error(traceback.format_stack())\n']
2023-03-31 15:19:06,466 ERROR [mpl] [MPListener.py:55] - FdcEqpModuleResDto.__init__() got an unexpected keyword argument 'detail'
2023-03-31 15:19:06,466 ERROR [mpl] [MPListener.py:56] - ['  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydev_bundle\\pydev_monkey.py", line 795, in __call__\n    ret = self.original_func(*self.args, **self.kwargs)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 995, in _bootstrap\n    self._bootstrap_inner()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 1038, in _bootstrap_inner\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 975, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 341, in __receiver_loop\n    self.process_frame(f, frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 190, in process_frame\n    self.notify(frame_type, f)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 243, in notify\n    notify_func(frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Listener\\MPListener.py", line 56, in on_message\n    self.__logger.error(traceback.format_stack())\n']
2023-03-31 15:23:56,764 ERROR [mpl] [MPLProcessWorker.py:73] - FdcEqpModuleResDto.__init__() got an unexpected keyword argument 'detail'
2023-03-31 15:23:56,773 ERROR [mpl] [MPLProcessWorker.py:74] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 74, in mplPWorker\n    loggerMpl.error(traceback.format_stack())\n']
2023-03-31 15:31:30,668 ERROR [mpl] [MPLProcessWorker.py:73] - FdcEqpModuleResDto.__init__() got an unexpected keyword argument 'detail'
2023-03-31 15:31:30,678 ERROR [mpl] [MPLProcessWorker.py:74] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 74, in mplPWorker\n    loggerMpl.error(traceback.format_stack())\n']
2023-03-31 15:39:07,700 ERROR [mpl] [MPLProcessWorker.py:73] - FdcEqpModuleResDto.__init__() got an unexpected keyword argument 'detail'
2023-03-31 15:39:07,709 ERROR [mpl] [MPLProcessWorker.py:74] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 74, in mplPWorker\n    loggerMpl.error(traceback.format_stack())\n']
2023-03-31 15:41:37,984 ERROR [mpl] [MPLProcessWorker.py:73] - FdcEqpModuleResDto.__init__() got an unexpected keyword argument 'detail'
2023-03-31 15:41:37,991 ERROR [mpl] [MPLProcessWorker.py:74] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 74, in mplPWorker\n    loggerMpl.error(traceback.format_stack())\n']
2023-03-31 16:02:41,983 ERROR [mpl] [MPLProcessWorker.py:73] - FdcEqpModuleResDto.__init__() got an unexpected keyword argument 'detail'
2023-03-31 16:02:41,986 ERROR [mpl] [MPLProcessWorker.py:74] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 74, in mplPWorker\n    loggerMpl.error(traceback.format_stack())\n']
2023-03-31 16:02:48,852 ERROR [mpl] [MPEqp.py:31] - bFdcAPI.Eqp.Dto.FdcEqpModule.FdcEqpModuleResDto() argument after ** must be a mapping, not str
2023-03-31 16:02:48,863 ERROR [mpl] [MPEqp.py:32] - ['  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydev_bundle\\pydev_monkey.py", line 795, in __call__\n    ret = self.original_func(*self.args, **self.kwargs)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 995, in _bootstrap\n    self._bootstrap_inner()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 1038, in _bootstrap_inner\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 975, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 341, in __receiver_loop\n    self.process_frame(f, frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 190, in process_frame\n    self.notify(frame_type, f)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 243, in notify\n    notify_func(frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Listener\\MPListener.py", line 53, in on_message\n    self.__mpListenerWorker.onCommandMessage(frame.body)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPListenerWorker.py", line 52, in onCommandMessage\n    self.deleteEqp(r.get("Eqp"), r.get("EqpCode"))\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPListenerWorker.py", line 57, in deleteEqp\n    for module in mEqp.getModules():\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPEqp.py", line 32, in getModules\n    self.__loggerMpl.error(traceback.format_stack())\n']
2023-03-31 16:04:50,887 ERROR [mpl] [MPEqp.py:31] - bFdcAPI.Eqp.Dto.FdcEqpModule.FdcEqpModuleResDto() argument after ** must be a mapping, not str
2023-03-31 16:04:50,904 ERROR [mpl] [MPEqp.py:32] - ['  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydev_bundle\\pydev_monkey.py", line 795, in __call__\n    ret = self.original_func(*self.args, **self.kwargs)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 995, in _bootstrap\n    self._bootstrap_inner()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 1038, in _bootstrap_inner\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 975, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 341, in __receiver_loop\n    self.process_frame(f, frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 190, in process_frame\n    self.notify(frame_type, f)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 243, in notify\n    notify_func(frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Listener\\MPListener.py", line 53, in on_message\n    self.__mpListenerWorker.onCommandMessage(frame.body)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPListenerWorker.py", line 52, in onCommandMessage\n    self.deleteEqp(r.get("Eqp"), r.get("EqpCode"))\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPListenerWorker.py", line 57, in deleteEqp\n    for module in mEqp.getModules():\n', '  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydevd_bundle\\pydevd_frame.py", line 884, in trace_dispatch\n    self.do_wait_suspend(thread, frame, event, arg)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydevd_bundle\\pydevd_frame.py", line 144, in do_wait_suspend\n    self._args[0].do_wait_suspend(*args, **kwargs)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\pydevd.py", line 1160, in do_wait_suspend\n    self._do_wait_suspend(thread, frame, event, arg, suspend_type, from_this_thread)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\pydevd.py", line 1174, in _do_wait_suspend\n    self.process_internal_commands()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\pydevd.py", line 873, in process_internal_commands\n    int_cmd.do_it(self)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydevd_bundle\\pydevd_comm.py", line 1847, in do_it\n    result, exception_occurred = pydevd_console_integration.console_exec(self.thread_id, self.frame_id, self.expression, dbg)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydevd_bundle\\pydevd_console_integration.py", line 235, in console_exec\n    Exec(code, updated_globals, updated_globals)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydevd_bundle\\pydevd_exec2.py", line 3, in Exec\n    exec(exp, global_vars, local_vars)\n', '  File "<input>", line 1, in <module>\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPEqp.py", line 32, in getModules\n    self.__loggerMpl.error(traceback.format_stack())\n']
2023-03-31 16:07:27,700 ERROR [mpl] [MPEqp.py:31] - bFdcAPI.Eqp.Dto.FdcEqpModule.FdcEqpModuleResDto() argument after ** must be a mapping, not str
2023-03-31 16:07:27,713 ERROR [mpl] [MPEqp.py:32] - ['  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydev_bundle\\pydev_monkey.py", line 795, in __call__\n    ret = self.original_func(*self.args, **self.kwargs)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 995, in _bootstrap\n    self._bootstrap_inner()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 1038, in _bootstrap_inner\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 975, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 341, in __receiver_loop\n    self.process_frame(f, frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 190, in process_frame\n    self.notify(frame_type, f)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\stomp\\transport.py", line 243, in notify\n    notify_func(frame)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Listener\\MPListener.py", line 53, in on_message\n    self.__mpListenerWorker.onCommandMessage(frame.body)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPListenerWorker.py", line 52, in onCommandMessage\n    self.deleteEqp(r.get("Eqp"), r.get("EqpCode"))\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPListenerWorker.py", line 57, in deleteEqp\n    for module in mEqp.getModules():\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPEqp.py", line 32, in getModules\n    self.__loggerMpl.error(traceback.format_stack())\n']
2023-03-31 16:10:35,113 ERROR [mpl] [MPLProcessWorker.py:73] - FdcEqpModuleResDto.__init__() got an unexpected keyword argument 'detail'
2023-03-31 16:10:35,117 ERROR [mpl] [MPLProcessWorker.py:74] - ['  File "<string>", line 1, in <module>\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 120, in spawn_main\n    exitcode = _main(fd, parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\spawn.py", line 133, in _main\n    return self._bootstrap(parent_sentinel)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 314, in _bootstrap\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\multiprocessing\\process.py", line 108, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 74, in mplPWorker\n    loggerMpl.error(traceback.format_stack())\n']
2023-03-31 17:04:55,890 ERROR [mpl] [BrokerConnect.py:62] - 
2023-03-31 17:04:55,910 ERROR [mpl] [BrokerConnect.py:63] - ['  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\pydevd.py", line 2195, in <module>\n    main()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\pydevd.py", line 2177, in main\n    globals = debugger.run(setup[\'file\'], None, None, is_module)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\pydevd.py", line 1489, in run\n    return self._exec(is_module, entry_point_fn, module_name, file, globals, locals)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\pydevd.py", line 1496, in _exec\n    pydev_imports.execfile(file, globals, locals)  # execute the script\n', '  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydev_imps\\_pydev_execfile.py", line 18, in execfile\n    exec(compile(contents+"\\n", file, \'exec\'), glob, loc)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\manage.py", line 22, in <module>\n    main()\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\manage.py", line 18, in main\n    execute_from_command_line(sys.argv)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\django\\core\\management\\__init__.py", line 446, in execute_from_command_line\n    utility.execute()\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\django\\core\\management\\__init__.py", line 420, in execute\n    django.setup()\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\django\\__init__.py", line 24, in setup\n    apps.populate(settings.INSTALLED_APPS)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\venv\\Lib\\site-packages\\django\\apps\\registry.py", line 124, in populate\n    app_config.ready()\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\apps.py", line 15, in ready\n    mplProcessWorker()\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\Process\\MPLProcessWorker.py", line 94, in mplProcessWorker\n    ActiveMqConnect(mpListenerWorker, coreInfo).connect()\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\BrokerConnect\\BrokerConnect.py", line 63, in connect\n    logging.getLogger("mpl").error(traceback.format_stack())\n']
2023-03-31 17:05:02,795 ERROR [mpl] [BrokerConnect.py:46] - 
2023-03-31 17:05:02,801 ERROR [mpl] [BrokerConnect.py:47] - ['  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydev_bundle\\pydev_monkey.py", line 795, in __call__\n    ret = self.original_func(*self.args, **self.kwargs)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 995, in _bootstrap\n    self._bootstrap_inner()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 1038, in _bootstrap_inner\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 975, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\BrokerConnect\\BrokerConnect.py", line 47, in reConnect\n    logging.getLogger("mpl").error(traceback.format_stack())\n']
2023-03-31 17:05:16,153 ERROR [mpl] [BrokerConnect.py:46] - 
2023-03-31 17:05:16,157 ERROR [mpl] [BrokerConnect.py:47] - ['  File "C:\\Users\\wisol\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\223.8214.51\\plugins\\python\\helpers\\pydev\\_pydev_bundle\\pydev_monkey.py", line 795, in __call__\n    ret = self.original_func(*self.args, **self.kwargs)\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 995, in _bootstrap\n    self._bootstrap_inner()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 1038, in _bootstrap_inner\n    self.run()\n', '  File "C:\\Users\\wisol\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\threading.py", line 975, in run\n    self._target(*self._args, **self._kwargs)\n', '  File "C:\\webpro\\pythonProject\\fdcmp\\mpl\\BrokerConnect\\BrokerConnect.py", line 47, in reConnect\n    logging.getLogger("mpl").error(traceback.format_stack())\n']
